package.js

```diff
@@ -1,8 +1,8 @@
 Package.describe({
-  name: 'tunguska:accounts-imgur',
+  name: 'apinf:accounts-fiware',
   version: '0.0.1',
-  summary: 'OAuth2 for Imgur',
-  git: 'https://github.com/robfallows/tunguska-accounts-imgur',
+  summary: 'OAuth2 for Fiware',
```

Comment:
@mauriciovieira i need the fiware logo to put in package, where i can find a nice image of fiware?